syncstorage/storage/sql/queries_spanner.py

```diff
@@ -0,0 +1,114 @@
+from sqlalchemy.sql import select, bindparam
```

Comment:
It's a little weird to have this one file of spanner-specific stuff over here under `storage/sql`, and then have the rest of the spanner backend in a different directory?  Is it worth making a `storage/spanner/` directory that can host them both?